src/main/scala/codacy/plugins/test/PluginsTests.scala

```diff
@@ -2,30 +2,32 @@ package codacy.plugins.test
 
 import java.nio.file.{Path, Paths}
 
-import codacy.utils.{FileHelper, Printer}
+import codacy.utils.{FileHelper}
 import com.codacy.analysis.core
 import com.codacy.analysis.core.model.{CodacyCfg, Issue, Pattern}
 import com.codacy.plugins.api.results.Result
 import com.codacy.plugins.results.traits.{DockerToolDocumentation, ToolRunner}
 import com.codacy.plugins.runners.{BinaryDockerRunner, DockerRunner}
 import com.codacy.plugins.utils.BinaryDockerHelper
+import better.files.File
 
 object PluginsTests extends ITest {
 
   val opt = "plugin"
 
-  def run(testSources: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
-    Printer.green("Running PluginsTests:")
+  def run(docsDirectory: File, dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)